[2024-03-15T00:41:23.296-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T04:38:00+00:00 [queued]>
[2024-03-15T00:41:23.303-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T04:38:00+00:00 [queued]>
[2024-03-15T00:41:23.303-0400] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-15T00:41:23.331-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): load_data_from_gcp> on 2024-03-15 04:38:00+00:00
[2024-03-15T00:41:23.342-0400] {standard_task_runner.py:60} INFO - Started process 142275 to run task
[2024-03-15T00:41:23.355-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'PII_Data_Detection', 'load_data_from_gcp', 'scheduled__2024-03-15T04:38:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/src/airflow.py', '--cfg-path', '/tmp/tmpdapff8wi']
[2024-03-15T00:41:23.362-0400] {standard_task_runner.py:88} INFO - Job 84: Subtask load_data_from_gcp
[2024-03-15T00:41:23.522-0400] {task_command.py:423} INFO - Running <TaskInstance: PII_Data_Detection.load_data_from_gcp scheduled__2024-03-15T04:38:00+00:00 [running]> on host MSI.
[2024-03-15T00:41:23.759-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='PII_Data_Detection' AIRFLOW_CTX_TASK_ID='load_data_from_gcp' AIRFLOW_CTX_EXECUTION_DATE='2024-03-15T04:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-15T04:38:00+00:00'
[2024-03-15T00:41:23.761-0400] {logging_mixin.py:188} INFO - environment path is set
[2024-03-15T00:41:23.761-0400] {logging_mixin.py:188} INFO - current project directory /home/vineshgvk/PII-Data
[2024-03-15T00:41:23.762-0400] {logging_mixin.py:188} INFO - destination directory at /home/vineshgvk/PII-Data/dags/processed
[2024-03-15T00:41:23.762-0400] {logging_mixin.py:188} INFO - Trying to create a storage client connection
[2024-03-15T00:41:24.117-0400] {logging_mixin.py:188} INFO - Trying to fetch the specified bucket name
[2024-03-15T00:41:25.093-0400] {logging_mixin.py:188} INFO - Specified bucket is successfully fetched
[2024-03-15T00:41:25.094-0400] {logging_mixin.py:188} INFO - Starting iteration on files from bucket
[2024-03-15T00:41:25.096-0400] {logging_mixin.py:188} INFO - defined the blob to fetch <Blob: pii_data_load, Data/train_com.json, None>
[2024-03-15T00:41:25.097-0400] {logging_mixin.py:188} INFO - local file path is created at /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T00:41:25.098-0400] {logging_mixin.py:188} WARNING - --- Logging error ---
[2024-03-15T00:41:25.100-0400] {logging_mixin.py:188} WARNING - Traceback (most recent call last):
[2024-03-15T00:41:25.101-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1100, in emit
    msg = self.format(record)
[2024-03-15T00:41:25.102-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 943, in format
    return fmt.format(record)
[2024-03-15T00:41:25.103-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 678, in format
    record.message = record.getMessage()
[2024-03-15T00:41:25.103-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 368, in getMessage
    msg = msg % self.args
[2024-03-15T00:41:25.104-0400] {logging_mixin.py:188} WARNING - TypeError: not all arguments converted during string formatting
[2024-03-15T00:41:25.104-0400] {logging_mixin.py:188} WARNING - Call stack:
[2024-03-15T00:41:25.112-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/bin/airflow", line 8, in <module>
    sys.exit(main())
[2024-03-15T00:41:25.113-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/__main__.py", line 57, in main
    args.func(args)
[2024-03-15T00:41:25.113-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
[2024-03-15T00:41:25.114-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 114, in wrapper
    return f(*args, **kwargs)
[2024-03-15T00:41:25.114-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 438, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
[2024-03-15T00:41:25.115-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 216, in _run_task_by_selected_method
    return _run_task_by_local_task_job(args, ti)
[2024-03-15T00:41:25.115-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 278, in _run_task_by_local_task_job
    ret = run_job(job=job_runner.job, execute_callable=job_runner._execute)
[2024-03-15T00:41:25.115-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
[2024-03-15T00:41:25.116-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/job.py", line 393, in run_job
    return execute_job(job, execute_callable=execute_callable)
[2024-03-15T00:41:25.116-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/job.py", line 422, in execute_job
    ret = execute_callable()
[2024-03-15T00:41:25.117-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/jobs/local_task_job_runner.py", line 165, in _execute
    self.task_runner.start()
[2024-03-15T00:41:25.117-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 48, in start
    self.process = self._start_by_fork()
[2024-03-15T00:41:25.118-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 100, in _start_by_fork
    ret = args.func(args, dag=self.dag)
[2024-03-15T00:41:25.118-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
[2024-03-15T00:41:25.119-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 114, in wrapper
    return f(*args, **kwargs)
[2024-03-15T00:41:25.119-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 438, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
[2024-03-15T00:41:25.120-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 218, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
[2024-03-15T00:41:25.120-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 297, in _run_raw_task
    return ti._run_raw_task(
[2024-03-15T00:41:25.120-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
[2024-03-15T00:41:25.121-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2360, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
[2024-03-15T00:41:25.121-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2532, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
[2024-03-15T00:41:25.122-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2549, in _execute_task
    return _execute_task(self, context, task_orig)
[2024-03-15T00:41:25.122-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
[2024-03-15T00:41:25.123-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
[2024-03-15T00:41:25.123-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
[2024-03-15T00:41:25.124-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
[2024-03-15T00:41:25.124-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/PII-Data/dags/src/data_download.py", line 63, in load_data_from_gcp_and_save_as_json
    logging.info('works', local_file_path)
[2024-03-15T00:41:25.125-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 2138, in info
    root.info(msg, *args, **kwargs)
[2024-03-15T00:41:25.125-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1477, in info
    self._log(INFO, msg, args, **kwargs)
[2024-03-15T00:41:25.126-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1624, in _log
    self.handle(record)
[2024-03-15T00:41:25.126-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1634, in handle
    self.callHandlers(record)
[2024-03-15T00:41:25.127-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 1696, in callHandlers
    hdlr.handle(record)
[2024-03-15T00:41:25.127-0400] {logging_mixin.py:188} WARNING -   File "/usr/lib/python3.10/logging/__init__.py", line 968, in handle
    self.emit(record)
[2024-03-15T00:41:25.128-0400] {logging_mixin.py:188} WARNING -   File "/home/vineshgvk/airflow_env/lib/python3.10/site-packages/airflow/utils/log/file_task_handler.py", line 272, in emit
    self.handler.emit(record)
[2024-03-15T00:41:25.128-0400] {logging_mixin.py:188} WARNING - Message: 'works'
Arguments: ('/home/vineshgvk/PII-Data/dags/processed/train_com.json',)
[2024-03-15T00:41:25.335-0400] {logging_mixin.py:188} INFO - File train_com.json downloaded and saved as /home/vineshgvk/PII-Data/dags/processed/train_com.json.
[2024-03-15T00:41:25.336-0400] {python.py:202} INFO - Done. Returned value was: /home/vineshgvk/PII-Data/dags/processed/train_com.json
[2024-03-15T00:41:25.374-0400] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=PII_Data_Detection, task_id=load_data_from_gcp, execution_date=20240315T043800, start_date=20240315T044123, end_date=20240315T044125
[2024-03-15T00:41:25.475-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-15T00:41:25.534-0400] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
